frontend/src/components/Graph/yAxisConfig.ts

```diff
@@ -1,86 +1,17 @@
 /* eslint-disable sonarjs/cognitive-complexity */
 import { formattedValueToString, getValueFormat } from '@grafana/data';
 import * as Sentry from '@sentry/react';
+import { UniversalYAxisUnit } from 'components/YAxisUnitSelector/types';
+import { isUniversalUnit } from 'components/YAxisUnitSelector/utils';
 import { isNaN } from 'lodash-es';
 
-const DEFAULT_SIGNIFICANT_DIGITS = 15;
```

Comment:
Had to move the data from here to separate utils and types file to avoid cyclic dependencies